ResearchKit/Common/ORKOrderedTask.m

```diff
@@ -296,14 +296,33 @@ - (id)initWithCoder:(NSCoder *)aDecoder {
 static NSString * const ORKInstruction0StepIdentifier = @"instruction";
 static NSString * const ORKInstruction1StepIdentifier = @"instruction1";
 static NSString * const ORKCountdownStepIdentifier = @"countdown";
+static NSString * const ORKCountdownAudioRecorderIdentifier = @"countdown.audio";
 static NSString * const ORKAudioStepIdentifier = @"audio";
+static NSString * const ORKAudioAudioRecorderIdentifier = @"audio.audio";
 static NSString * const ORKTappingStepIdentifier = @"tapping";
+static NSString * const ORKTappingAccelerometerRecorderIdentifier = @"tapping.accelerometer";
 static NSString * const ORKConclusionStepIdentifier = @"conclusion";
 static NSString * const ORKFitnessWalkStepIdentifier = @"fitness.walk";
+static NSString * const ORKFitnessWalkPedometerRecorderIdentifier = @"fitness.walk.pedometer";
+static NSString * const ORKFitnessWalkAccelerometerRecorderIdentifier = @"fitness.walk.accelerometer";
+static NSString * const ORKFitnessWalkDeviceMotionRecorderIdentifier = @"fitness.walk.deviceMotion";
+static NSString * const ORKFitnessWalkLocationRecorderIdentifier = @"fitness.walk.location";
```

Comment:
You make a very good point, I'm modifying this.

One question though, hypothetically couldn't several `ORKHealthQuantityTypeRecorder` be added to the same step (for different health units)?